### ISSUE TITLE
[Low] Loot quest starting item (BB #102)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#102]
This issue was migrated from bitbucket.
**Original Reporter:** stfx
**Original Date:** 08.04.2010 08:37:06 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/102

<hr>

On mobs which drop quest starting items ... and you pick one say http://www.wowhead.com/item=2839 and you dont start the quest. Then this quest starting item can still drop but you cant loot it. It should not drop if you 1. started quest (working) and 2. have item in bag (not working)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 09.01.2014 17:22:14 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/102/#comment-7656806

<hr>

```
#!c++

         ItemPrototype const *pProto = objmgr.GetItemPrototype(itemid);
         if (pProto && pProto->StartQuest && player->GetQuestStatus(pProto->StartQuest) != QUEST_STATUS_NONE && !player->HasQuestForItem(itemid))
             return false;
+
+       if (pProto && pProto->StartQuest && player->HasItemCount(itemid, 1, true))
+           return false;
     }

     return true;

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** release_ok
**Original Date:** 11.01.2014 08:57:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/102/#comment-7680633

<hr>

Resolved issue #102 - Loot Quest starting item:

Before this was resolved a player could pick up more then one item that starts
the same quest, eventually stacking piles of them.

Also cleaned up some code that I forgot to do in last commits.
Once again, this issue was resolved by LordUsagi.

→ <<cset 675789b53cb3>>
